docs/public-networks/reference/engine-api/index.md

```diff
@@ -224,6 +224,64 @@ curl -X POST --data '{"jsonrpc":"2.0","method":"engine_forkchoiceUpdatedV1","par
 
 </Tabs>
 
+### `engine_getBlobsV1`
+
+Returns the blobs corresponding to the specified blob versioned hashes.
+
+#### Parameters
+
+`blobHashes`: *array* of *strings* - Array of blob versioned hashes
+
+#### Returns
+
+`engineGetBlobsResultV1`: *array* of *objects* - Array of blob and proof objects
```

Comment:
Do we want to mention that for versioned hashes where the blob was not available besu responds with a null at that index of the response? In the spec they say this:
* result: `Array of BlobAndProofV1` - Array of [`BlobAndProofV1`](#BlobAndProofV1), items of which may be `null`.

Reply:
In our other API documentation do we use the 'official' names of the parameters and results? The result elements are called BlobAndProofV1.

Reply:
For the type, using the generic "array of objects" is a little easier to understand. I've added a section detailing the blob and proof object for more information.